raiden/blockchain/events.py

```diff
@@ -144,7 +147,7 @@ def get_all_netting_channel_events(
         pyethapp_chain,
         netting_channel_address,
         events=ALL_EVENTS,
-        from_block=0,
+        from_block='earliest',
```

Comment:
Hmmm I would say make the default then the block 0. `earliest` is not explicit enough. I don't doubt your empirical evidence as I just checked another documentation [here](https://ethereumbuilders.gitbooks.io/guide/content/en/ethereum_json_rpc.html) which seems to suggest that this is indeed the earliest/genesis block. But if we have it at block `0` we explicitly request the earliest block and don't rely on client implementations of this special string or don't get confused by not up to date documentation.